src/main/java/org/broadinstitute/hellbender/tools/exome/SegmentUtils.java

```diff
@@ -451,6 +536,41 @@ private static ModeledSegment toModeledSegment(final DataLine dataLine, final bo
         }
     }
 
+    /*===============================================================================================================*
+     * PRIVATE METHODS FOR READING/WRITING SEGMENT FILES                                                             *
+     *===============================================================================================================*/
+
+
+    private static ModeledSegment toModeledSegment(final DataLine dataLine, final boolean takeLog2) {
+        final double mean = dataLine.getDouble(SegmentTableColumns.MEAN.toString());
+
+        return new ModeledSegment(
+                toInterval(dataLine),
+                dataLine.get(SegmentTableColumns.CALL.toString(), ModeledSegment.NO_CALL),
+                dataLine.getLong(SegmentTableColumns.NUM_PROBES.toString()),
+                takeLog2 ? ParamUtils.log2(mean) : mean);
+    }
+
+    private static ACNVModeledSegment toACNVModeledSegment(final DataLine dataLine) {
+
+        final PosteriorSummary segmentMeanPosteriorSummary = new PosteriorSummary(dataLine.getDouble(SegmentTableColumns.SEGMENT_MEAN_POSTERIOR_MODE.toString()),
+                dataLine.getDouble(SegmentTableColumns.SEGMENT_MEAN_POSTERIOR_LOWER.toString()),
+                dataLine.getDouble(SegmentTableColumns.SEGMENT_MEAN_POSTERIOR_UPPER.toString()));
+        final PosteriorSummary minorAlleleFractionPosteriorSummary = new PosteriorSummary(dataLine.getDouble(SegmentTableColumns.MINOR_ALLELE_FRACTION_POSTERIOR_MODE.toString()),
+                dataLine.getDouble(SegmentTableColumns.MINOR_ALLELE_FRACTION_POSTERIOR_LOWER.toString()),
+                dataLine.getDouble(SegmentTableColumns.MINOR_ALLELE_FRACTION_POSTERIOR_UPPER.toString()));
+
+        return new ACNVModeledSegment(
+                toInterval(dataLine),
+                segmentMeanPosteriorSummary,
+                minorAlleleFractionPosteriorSummary);
+    }
+
+    private static String toSampleName(final DataLine dataLine) {
+        return dataLine.get(SegmentTableColumns.SAMPLE.toString());
+    }
+
+
```

Comment:
whitespace.  Also see comment in ACSModeledSegmentUtils about making writeSegmentFile public.